batch_job/src/batch_job_condor.c

```diff
@@ -128,10 +128,19 @@ static batch_job_id_t batch_job_condor_submit (struct batch_queue *q, const char
 	fprintf(file, "keep_claim_idle = 30\n");
 	fprintf(file, "log = %s\n", q->logfile);
 
+	const char *c_req = batch_queue_get_option(q, "condor-requirements");
 	char *bexp = blacklisted_expression(q);
-	if(bexp) {
-		fprintf(file, "requirements = %s\n", bexp);
-		free(bexp);
+    if(c_req) {
```

Comment:
The autobuild email you got had some advice on how to deal with this:

Checking suitability for merging:
        Whitespace check... failed.
                Fix using: git rebase --whitespace=fix $(git merge-base master HEAD)
                git diff --check master...c4de028f5e201fed27cc3d5b2d627389ec482c71
                        batch_job/src/batch_job_condor.c:133: indent with spaces.
                        +    if(c_req) {
                        batch_job/src/batch_job_condor.c:134: indent with spaces.
                        +        if(bexp) {
                        batch_job/src/work_queue_factory.c:726: indent with spaces.
                        +            case LONG_OPT_CONDOR_REQUIREMENTS:
                        batch_job/src/work_queue_factory.c:728: indent with spaces.
                        +                // will implement
                        batch_job/src/work_queue_factory.c:729: space before tab in indent.
                        +                               break;
                        batch_job/src/work_queue_factory.c:845: indent with spaces.
                        +    if (condor_requirements != NULL) {
                        batch_job/src/work_queue_factory.c:847: indent with spaces.
                        +    }